randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -476,11 +479,11 @@ Extra - asset_id: 13
   > Door to Gold Four Hallway
       Any of the following:
           # Weapons to defeat Green Pirates
-          Charge Beam or Ice Beam or Missile or Plasma Beam or Power Bombs or Spazer Beam or Super Missile or Wave Beam or Can Lay Any Bombs
-          # 3 hits for Intermediate damage boosting
-          Damage Boosting (Intermediate) and Normal Damage ≥ 61
+          Charge Beam or Ice Beam or Missile or Plasma Beam or Spazer Beam or Super Missile or Wave Beam or Can Lay Any Bombs
           # 5 hits for beginner damage boosting
           Damage Boosting (Beginner) and Normal Damage ≥ 101
+          # 3 hits for Intermediate damage boosting
+          Damage Boosting (Intermediate) and Normal Damage ≥ 61
 
```

Comment:
There should also be a connection from Door to Terminator Room to Gauntlet Tunnel Bottom. You can't get up to the items, but if you put a node here you can reach it. Its blocked by powerbombs. Its often used to look at the items before deciding to go down the gauntlet.
![image](https://github.com/randovania/randovania/assets/46821362/374790da-d8c5-4f32-b048-0d82f745d860)

Reply:
Ehhhhhh I feel like this isnt necessary to add considering that this doesnt add anything logical. Neither generator nor solver have a concept of "peeking"

Reply:
I don't have a problem with that either. Just for the sake of completion I thought I'd mention it.